client/my-sites/site-settings/disconnect-site/disconnect-survey.jsx

```diff
@@ -0,0 +1,96 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import CompactCard from 'components/card/compact';
+import { getSelectedSite, getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import { isFreeJetpackPlan } from 'lib/products-values';
+import { isJetpackSite } from 'state/sites/selectors';
+
+class DisconnectSurvey extends Component {
+	state = {
+		reasonSelected: 'onlyNeedFree',
```

Comment:
If I'm following this correctly, the initial state selects a question which may not be included. Could that be problematic?